templates/manila/config/manila-scheduler-config.json

```diff
@@ -6,6 +6,12 @@
       "dest": "/etc/manila/manila.conf.d",
       "owner": "root:manila",
       "perm": "0750"
+    },
+    {
```

Comment:
@gouthampacha we only copy this file on manila-scheduler and manila-share, but not in manila-api that doesn't use this logging mechanism.

Reply:
++